src/shared/components/alert-button/AlertButton.test.tsx

```diff
@@ -0,0 +1,139 @@
+/**
```

Comment:
Hmm, I think this whole test (as well as the one in the QuestionButton) now belongs to the `useShowTooltip` hook. It's the hook's logic that's being tested.